### ISSUE TITLE
Font Size Needs to be Tied to Width, Height, or Both of Container and Not VMIN or EM, etc.

### ISSUE THREAD

**Issue by r-neal-kelly** [r-neal-kelly/triple_triad#15]
Now that we have the ability to size everything to exact pixels (including the fraction of pixel) we can actually do our own proportions for the pixel size of fonts.

**Comment by r-neal-kelly** [comment]
29a8987bcfbdd5126bff5efb1804603c52f308a7 partially addresses this issue by making player names scroll

**Comment by r-neal-kelly** [comment]
Closing this for now as it seems the only outstanding font issue is correctly measuring the bumper size to make sure it can fit what is there.

**Comment by r-neal-kelly** [comment]
See issue #24

**Comment by r-neal-kelly** [comment]
Forgot about the popups for cells not fitting correctly with large boards, so we still need to address this issue separately from #24, although it's related.

**Comment by r-neal-kelly** [comment]
Well we now have the functionality in utils to achieve this not just for width but for height, and even for both at the same time. The only problem, even though we have made it efficient by using proportions to cut down on iterations, having the browser draw text is incredibly expensive in any situation, and doing so in a loop can certainly be worse.

However, having tested it, and we should add long-term tests in the test suite to verify this repeatedly, I've found that our algorithm nets us the fitted font size in about 4-5 draw calls so it's not bad. However we want to do as few of those as possible and only at designated times in the life of the program (life and resize events).

It should be reasonable to do 1 for the += symbol on popups and the Combo symbol on popups, which are the last remaining sore-eye spot in the program regarding text. It would be nice to do all place that text occurs outside of prose, but let's get the key areas before overloading the browser with font draw calls.
